File: core/core-api/src/main/java/com/coffee/gu/store/StoreService.java
package com.coffee.gu.store;

import org.springframework.stereotype.Service;

import java.util.List;

@Service
public class StoreService {

    private final StoreReader storeReader;

    public StoreService(StoreReader storeReader) {
        this.storeReader = storeReader;
    }

    public List<Store> getAroundStores(StoreSearch storeSearch) {
        return storeReader.getAroundStores(storeSearch);
    }
}

File: core/core-api/src/main/kotlin/com/coffee/gu/store/StoreService.kt
package com.coffee.gu.store

import org.springframework.stereotype.Service

@Service
class StoreService(
    private val storeReader: StoreReader,
) {
    fun getAroundStores(storeSearch: StoreSearch): List<Store> {
        return storeReader.getAroundStores(storeSearch)
    }
}
